js.js

```diff
@@ -26,15 +44,15 @@ const addBooks = () => {
     return;
   }
 
-  const newBook = {
-    title,
-    author,
-  };
-
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
+  const newBook = new Book(title, author).addtolist();
+  bttnn.addEventListener('click', newBook);
   outputbooklist();
 };
+function removefromlist(index) {
+  const newBook = new Book().removefromlist(index);
+  bttnn.addEventListener('click', newBook);
```

Comment:
- [ ] OPTIONAL: This function is throwing a console error. Kindly incorporate error handling here as your app needs to be free of errors even though the app is working. :pray: In addition, I don't think you need to initialize a new book object as you remove a book from a list using its index. :pray: